src/core/core.controller.js

```diff
@@ -147,8 +147,8 @@ function updateConfig(chart) {
 	chart.tooltip.initialize();
 }
 
-function positionIsHorizontal(position) {
-	return position === 'top' || position === 'bottom';
+function positionIsHorizontal(position, axis = undefined) {
+	return position === 'top' || position === 'bottom' || axis === 'x';
```

Comment:
Could we get this from the `scaleId`?  I know we had talked about introducing this anyway at some point, but just as we've done elsewhere it seems like it shouldn't be necessary if the scaleId is `x`

Reply:
We could. I'm ok to do that here to keep this simpler and then add an `axis` option in a separate PR.

Reply:
I think it'd keep it'd make it easier for the user because then they wouldn't need to specify `axis` most of the time and it'd be more consistent that specifying `scales.x` means it's an x scale everywhere

Reply:
So, I had modified it to determine the axis from the ID. That causes problems when we change the positions of an axis because the axis configs have existing defaults. I'm going to try removing the defaults entirely for axis positions, and then maybe this will be cleaned up